src/lib/site-import/status.js

```diff
@@ -236,17 +237,52 @@ ${ maybeExitPrompt }
 				} catch ( error ) {
 					return reject( { error } );
 				}
-				const { importStatus, importJob } = status;
+				const { importStatus } = status;
+				let { importJob } = status;
+
+				let jobStatus, jobSteps = [];
+				if ( env.isK8sResident ) {
+					// in the future the API may provide this in k8s jobs so account for that.
+					// Until then we need to create the importJob from the status object.
+					if ( ! importJob ) {
```

Comment:
This _should_ allow us to send this object from the server in the future without requiring an update on the client.